tests/tests_fabric/utilities/test_throughput_monitor.py

```diff
@@ -0,0 +1,94 @@
+from unittest import mock
+from unittest.mock import Mock, call
+
+import torch
+from lightning.fabric import Fabric
+from lightning.fabric.utilities.throughput_monitor import ThroughputMonitor, measure_flops
+
+from tests_fabric.helpers.runif import RunIf
+from tests_fabric.test_fabric import BoringModel
+
+
+@RunIf(min_torch="2.1")
+def test_measure_flops():
+    with torch.device("meta"):
+        model = BoringModel()
+        x = torch.randn(2, 32)
+    model_fwd = lambda: model(x)
+    model_loss = lambda y: y.sum()
+
+    training_flops = measure_flops(model, model_fwd, model_loss)
```

Comment:
why isn't this using `model.forward` directly? Why is a lambda needed?
(same question for the docstring in measure_flops)

Reply:
Because you need to specify how the batch is forward-ed. An alternative would be to take `*forward_args, **forwar_kwargs` in `measure_flops`.

Reply:
I think that would still leave the question how to handle the loss. Resolving for now